src/worldmap.test.ts

```diff
@@ -370,6 +370,7 @@ describe('Worldmap', () => {
         mapCenterLongitude: 0,
         initialZoom: 1,
         colors: ['red', 'blue', 'green'],
+        replaceVariables: (interpolation: string) => {}
       },
```

Comment:
Don't you want to cover the new / adjusted functionality with tests?

Reply:
Well, there were no tests for the whole thing yet. In fact, the plugin itself has a very low test coverage. We should update for the new plugin version, move our custom stuff there and then upgrade the tests.